src/literal.cc

```diff
@@ -27,11 +27,11 @@ namespace wabt {
 namespace {
 
 int Clz(uint32_t value) {
-  return wabt_clz_u32(value);
+  return value != 0 ? wabt_clz_u32(value) : 32;
```

Comment:
Do you think its a little easer to read the other way around: `value == 0 ? 32 : wabt_clz_u32(value);`